<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AutorisationDialog</class>
 <widget class="QDialog" name="AutorisationDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>195</width>
    <height>123</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>195</width>
    <height>123</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="page">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QComboBox" name="ComboBox_Auth_User"/>
       </item>
       <item>
        <widget class="QPushButton" name="PButton_Auth_Begin">
         <property name="text">
          <string>Begin new test</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="PButton_Auth_Admin">
         <property name="text">
          <string>PushButton</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_2">
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLineEdit" name="LEdit_Login_Login">
         <property name="placeholderText">
          <string>Login</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="LEdit_Login_Password">
         <property name="placeholderText">
          <string>Password</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="PButton_Auth_Login">
         <property name="text">
          <string>Login</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
